### ISSUE TITLE
[Glue] Add Ability to Update User Profile Picture

### ISSUE THREAD

**Issue by jcsison** [jcsison/dealicious#36]
## Description
Depends on #31 and #35. The user should be able to update their profile picture from the user settings page. Clicking on the "Edit" button should bring up a modal that allows the user to input a URL for their profile picture (actual picture uploading will be deferred to whenever we have cloud storage integrated).

## Acceptance Criteria
- A modal with an input for the user to enter in their profile picture URL appears when the "Edit" button is clicked on in the user settings page
- The modal contains a submit button that updates the user's profile picture URL in the backend while also displaying a loading indicator when clicked on
- The user's profile picture in the settings page and in the navbar is updated as soon as the API call is finished
